app-frontend/src/app/components/pages/project/createAnalysis/index.js

```diff
@@ -340,25 +340,23 @@ class ProjectCreateAnalysisPageController {
             .open({
                 component: 'rfFeedbackModal',
                 resolve: {
-                    title: () => `No AOI defined for "${layer.name}"`,
+                    title: () => 'No AOI defined',
                     content: () =>`
                         <h2>
                             Creating an analyses requires an AOI
                         </h2>
                         <p>
-                            but the layer "${layer.name}" does not have an AOI defined.
+                            At least one selected layer does not have an AOI defined.
+                            Click on the missing AOI warning icon
+                            (<i class="icon-warning color-danger"></i>)
+                            on selected layers to define an AOI.
```

Comment:
I don't think you need to specify "on the selected layers" since this applies to all layers. Also, since you've specified "At least on selected layer", restating that you should be clicking the warning icon on selected layers afterwards is a bit redundant.